### ISSUE TITLE
[BUG] 0.7.1 升级 0.8.0

### ISSUE THREAD

**Issue by SturdyCat** [juicedata/juicefs-csi-driver#264]
```
helm upgrade --history-max=5 --install=true --namespace=kube-system --timeout=10m0s --values=/home/shell/helm/values-juicefs-csi-driver-0.8.0.yaml --version=0.8.0 --wait=true juicefs-csi-driver-0-1640586654 /home/shell/helm/juicefs-csi-driver-0.8.0.tgz
--
Error: UPGRADE FAILED: YAML parse error on juicefs-csi-driver/templates/storageclass.yaml: error converting YAML to JSON: yaml: line 21: could not find expected ':'
```

**Environment**:
- JuiceFS CSI Driver version (which image tag did your CSI Driver use): helm  juicefs-csi-driver-0.7.1
- Kubernetes version (e.g. `kubectl version`):  Client v1.23.1 Server v1.21.7
- Object storage (cloud provider and region): minio
- Metadata engine info (version, cloud provider managed or self maintained): mysql
- Network connectivity (JuiceFS to metadata engine, JuiceFS to object storage):
- Others:

**Comment by zwwhdls** [comment]
Thanks for feedback!

**Comment by zwwhdls** [comment]
hi @SturdyCat , chart 0.8.1 has released, you can upgrade to 0.8.1
